### ISSUE TITLE
Missing API method documentation

### ISSUE THREAD

**Issue by weikang9009** [martibosch/swisslandstats-geopy#3]
I would suggest some additional work on API documentation for the general public (sth like [the API references page of seaborn](https://seaborn.pydata.org/api.html)). A documentation website based on [readthedocs](https://readthedocs.org/) would be great.

This issue is related to the review of the JOSS submission https://github.com/openjournals/joss-reviews/issues/1511
`Functionality documentation: Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?`

**Comment by martibosch** [comment]
Hello,
the main functionalities are documented as docstrings of their respective methods. Nonetheless, I have compiled them via Sphinx into a [dedicated readthedocs website](https://swisslandstats-geopy.readthedocs.io/). I hope that this addresses the issue.

Martí

**Comment by weikang9009** [comment]
Hi Martí, the documentation website looks nice, but it seems you only expose `LandDataFrame` class on the website https://swisslandstats-geopy.readthedocs.io/en/latest/swisslandstats.html while other functions (like [`read_csv` function](https://github.com/martibosch/swisslandstats-geopy/blob/master/swisslandstats/dataframe.py)) are not listed on the docs website?

**Comment by martibosch** [comment]
Hello Wei,
I have amended the latest commit and included the documentation for the `merge` and `read_csv` functions. I have ommited creating documentation pages for the `plotting` and `geometry` modules on purpose, since all their respective functions are embedded as methods of the `LandDataFrame` class, and therefore their documentation can already be found there. I believe that such modularization of the library is appropriate for the maintainability of the code but does not concern so much the end user (the main target for whom the documentation is provided).
